Makefile

```diff
@@ -466,6 +466,16 @@ fv/fv.test: vendor/.up-to-date $(SRC_FILES)
 	$(DOCKER_GO_BUILD) go test ./$(shell dirname $@) -c --tags fvtests -o $@
 
 .PHONY: fv
+# runs all of the fv tests
+# to run it in parallel, decide how many parallel engines you will run, and in each one call:
+#         $(MAKE) fv FV_BATCHES_TO_RUN="<num>" FV_NUM_BATCHES=<num>
+# where
+#         FV_NUM_BATCHES = total parallel batches
+#         FV_BATCHES_TO_RUN = which number this is
+# e.g. to run it in 10 parallel runs:
+#         $(MAKE) fv FV_BATCHES_TO_RUN="1" FV_NUM_BATCHES=10     # the first 1/10
+#         $(MAKE) fv FV_BATCHES_TO_RUN="1" FV_NUM_BATCHES=10     # the next 1/10
```

Comment:
Typo: `FV_BATCHES_TO_RUN="1"` (should be 2 on this line)

Reply:
Good point, will fix.